### ISSUE TITLE
Uploading form containing duplicate properties (with different capitalization) - information in the warning

### ISSUE THREAD

**Issue by dbemke** [getodk/central#845]
#### Problem description
If a user tries to add a form or a new version of a form containing duplicate properties there’s the warning “The entity definition within the form is invalid. Multiple Form Fields cannot be saved to a single property.” 
If a new version of a form tries to add the duplicate property the users sees more information (which property cannot be added). 
#### Steps to reproduce the problem
1. In “Forms” tab click “+New” and upload the form containing 2 properties (with different capitalization).
[movie entities (1).xlsx.txt](https://github.com/user-attachments/files/18093634/movie.entities.1.xlsx.txt)
#### Screenshot

![sameporpertisInFrom](https://github.com/user-attachments/assets/b1355806-490c-4a97-81fc-14698694738a)

#### Expected behavior
There could be more information about the duplicate property in all warnings.
#### Central version shown in version.txt

https://staging.getodk.cloud/
versions:
20878c7ffe3b92a2b9f35bb253e6043e5ddba67b (v2024.2.1-11-g20878c7)
+c9b8bef4652436d7d055d8777d9e84966ecba661 client (v2024.2.1-57-gc9b8bef4)
+313da04aaa3133978a3ef31dcc3cdc63bbbc152a server (v2024.2.0-118-g313da04a)
https://test.getodk.cloud/
versions:
801ab503c74eaa24b27a226622eceee9351d5db0 (v2024.2.1-10-g801ab50)
+5e95312bc6177ed79f170d8851da39fe327219d4 client (v2024.2.1-63-g5e95312b)
+4355d9e4c3a0f7bb6b694aad4cf318145905da2e server (v2024.2.0-114-g4355d9e4)

**Comment by matthew-white** [comment]
I agree that it'd be helpful to list more information in this error message. :+1: It should also be translated (#489). It sounds like this is a preexisting issue, so I think we can wait to work on it until after v2024.3 is released.

**Comment by matthew-white** [comment]
Just to be clear, when we solve this issue, I think we would do so for all cases in which the error message is shown. Not just in the case where there are properties whose names are duplicates except for case.
